pkg/alertmanager/multitenant.go

```diff
@@ -327,17 +328,25 @@ func (am *MultitenantAlertmanager) transformConfig(userID string, amConfig *amco
 	if am.cfg.AutoSlackRoot != "" {
 		for _, r := range amConfig.Receivers {
 			for _, s := range r.SlackConfigs {
-				if s.APIURL == autoSlackURL {
-					s.APIURL = amconfig.Secret(am.cfg.AutoSlackRoot + "/" + userID + "/monitor")
+				if s.APIURL.String() == autoSlackURL {
```

Comment:
I suspect Weaveworks was the only user of this code and we've moved to just webhook, so we could remove it.